scripts/globals/effects/crooked_cards.lua

```diff
@@ -0,0 +1,16 @@
+-----------------------------------
+--
+--     tpz.effect.CROOKED_CARDS Increases the effects of the next phantom roll.	
```

Comment:
Minor inconsistency in the comments: description is on the same line as the effect name.

Reply:
removed note from descrip